src/main/java/org/apache/mesos/hdfs/util/HDFSConstants.java

```diff
@@ -22,4 +22,8 @@
   public static final String NODE_EXECUTOR_ID = "NodeExecutor";
   public static final String NAME_NODE_EXECUTOR_ID = "NameNodeExecutor";
 
+  // Path to Store HDFS Binary
+  public static final String MESOSPHERE_OPT_PATH = "/opt/mesosphere/";
```

Comment:
Please make this configurable, and call it something more generic/purpose-driven, like HDFS_MOUNT_PATH, remove HDFS_BINARY_PATH as it should be included.